src/Pmi/Service/PatientStatusService.php

```diff
@@ -0,0 +1,65 @@
+<?php
+namespace Pmi\Service;
+
+use Pmi\Audit\Log;
+use Pmi\PatientStatus\PatientStatus;
+
+class PatientStatusService
+{
+    protected $app;
+    protected $db;
+    protected $rdr;
+
+    public function __construct($app)
+    {
+        $this->app = $app;
+        $this->db = $app['db'];
+        $this->em = $app['em'];
+        $this->rdr = $app['pmi.drc.participants'];
+    }
+
+    public function sendPatientStatusToRdr()
+    {
+        $limit = $this->app->getConfig('patient_status_queue_limit');
+        $patientStatuses = $this->em->getRepository('patient_status')->fetchBySql("id not in (select patient_status_id from rdr_patient_status_log) limit 0, $limit");
```

Comment:
If there isn't an offset, just put the limit.  No need for `0, `

Reply:
I think I would prefer left joining to the rdr_patient_status_log and looking for nulls. Probably not a big deal either way, especially if you have a reason for doing it this way.